lib/services/notification-api-service.js

```diff
@@ -70,14 +72,47 @@ function notificationApiServiceFactory(
         });
     };
 
-    notificationApiService.prototype.postBroadcastNotification = function(message) {
-        var self = this;
-
+    NotificationApiService.prototype.postBroadcastNotification = function(message) {
         return eventsProtocol.publishBroadcastNotification(message)
         .then(function () {
             return message;
         });
     };
 
-    return new notificationApiService();
+    NotificationApiService.prototype.postProgressEvent = function(data) {
```

Comment:
I would suggest to change function name to ```postProgressNotification```  to be better aligned with previous function.  This is not a big deal anyway.